src/mint-release.ts

```diff
@@ -75,7 +75,7 @@ export class MintRelease {
     const updates: Update[] = [];
 
     updates.push(new Changelog({
-      path: 'CHANGELOG.md',
+      path: 'CHANGELerG.md',
```

Comment:
heh, wat?

Reply:
whoops, was testing files that don't yet exist, reverted shortly.